XCUITests/IntegrationTests.swift

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import XCTest
+
+class IntegrationTests: BaseTestCase {
+
+    func allowNotifications () {
+        addUIInterruptionMonitor(withDescription: "notifications") { (alert) -> Bool in
+            alert.buttons["Allow"].tap()
+            return true
+        }
+        sleep(5)
+        app.swipeDown()
+    }
+
+    private func bookmark() {
+        navigator.goto(PageOptionsMenu)
+        waitforExistence(app.tables.cells["Bookmark This Page"])
+        app.tables.cells["Bookmark This Page"].tap()
+        navigator.nowAt(BrowserTab)
+    }
+
+    func testFxASyncBookmark () {
```

Comment:
We may need to create a different XCUITest schema for the sync tests and have the tests enabled only there, otherwise this test is going to fail whenever we launch the XCUITests build on BB. And, if we disable it in our schemas to avoid the failure, then it fails when it is launched with `pipenv run pytest`

Reply:
This is what I did originally, but I thought perhaps it was excessive. I can look into making this change, however I'm PTO until Tuesday. Feel free to push followup commits to my branch if you want to get to this before then.